CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+3.21.3:
+    - Guard against /sys/hypervisor/uuid not being readable (ENT-9931)
```

Comment:
@craigcomstock I don't think your changelog entry conveys what was changed (for an end user). Here's my suggestion:

```suggestion
    - Fixed 'regline' error messages which appear on systems (ex. CentOS 6) where /sys/hypervisor/uuid is not readable (ENT-9931)
```